wiki/People/Beatmap_Nominators/en.md

```diff
@@ -25,11 +27,11 @@ Modders that aim to apply to the Beatmap Nominators must fulfil the following cr
 + They provide 2 to 4 of their mods which they think qualify them to be a part of the Beatmap Nominators.
 
 ### What Qualifies as a Mod Post
-+ Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April, the mod posts for January, February, and March will be relevant. The relevant months will also be listed on the submission form.
-+ A mod post qualifies as one if it has received a Kudosu and has been created within the time span listed on the submission form.
++ Upon signing up the website will automatically calculate your kudosu score from the last 3 months.
++ A mod post qualifies as one if it has received a Kudosu and has been created within the 3 months.
 + Since November 2017, beatmaps affected under Modding V2 had a reward system implemented based on upvotes/downvotes. The "mod post" equivalent in this case is a beatmap discussion, in which at least one suggestion/problem got awarded with an upvote.
 + The gamemode (osu!, osu!mania, osu!catch, osu!taiko) that you modded beatmaps in does not matter as your activity in the modding community as a whole is being measured and not the activity in a certain game mode.
-+ The point in time at which the Kudosu has been awarded to you mod post does not matter. The only relevant information about a mod post is its post date and time in UTC+0 as well as the fact that it has been awarded Kudosu.
++ The point in time at which the Kudosu has been awarded to your mod post **does** matter. 
```

Comment:
Add up something like ", as the calculator embed in the website will fetch the data from your own Modding History evens" for further clarification.